RULES_DESCRIPTIONS.md

```diff
@@ -475,12 +475,13 @@ Available flags are:
 * _disableStutteringCheck_ disables checking for method names that stutter with the package name (i.e. avoid failure messages of the form _type name will be used as x.XY by other packages, and that stutters; consider calling this Y_)
 * _sayRepetitiveInsteadOfStutters_ replaces the use of the term _stutters_ by _repetitive_ in failure messages
 * _checkPublicInterface_ enabled checking public method definitions in public interface types
+* []string list of code types to ignore warnings, avaliable options: function, method, type and var
 
 Example:
 
 ```toml
 [rule.exported]
-  arguments = ["checkPrivateReceivers", "disableStutteringCheck", "checkPublicInterface"]
+  arguments = ["checkPrivateReceivers", "disableStutteringCheck", "checkPublicInterface", ["function","method","type","var"]]
```

Comment:
the configuration entry gives no clue about the "ignore" nature of the list.

Reply:
ok i try struct with dedicated fields but in toml there is problem with notation like:

``
IgnoredWarnings{field:true,field2:true}
``

you can only use like this:

``
{field:true,field2:true} or list ["val","val2","val3"]
``

but there is no option to name/prefix that (or maybe I dont know the magic yet :()

Reply:
I think what @chavacava is asking is to have a better naming of the setting, e.g. `disableFunctionDocumentationCheck` instead of `function`...

Reply:
> I think what @chavacava is asking is to have a better naming of the setting, e.g. `disableFunctionDocumentationCheck` instead of `function`...

ok so each element of ["function","method","type","var"] as dedicated flag 

arguments = ["disableFunctionCheck", "disableMethodCheck", "disableTypeCheck", "disableVarCheck"]

that was my first idea, but thinks using array/slice will be simpler (one place to modify) :)

Reply:
ok maybe we can walk that way:

arguments = ["disableFunctionCheck", "disableMethodCheck", "disableTypeCheck", "disableVarCheck"]

and of course modify my commited code to use that flags insted of additional array in config what you think @chavacava ?